sandpack-client/package.json

```diff
@@ -31,7 +35,7 @@
     "prebuild": "yarn run clean",
     "test": "jest .",
     "lint": "eslint '**/*.ts?(x)' --fix",
-    "build": "node build.js && yarn run build:types",
+    "build": "rollup -c --bundleConfigAsCjs && yarn run build:types",
```

Comment:
Maybe we can let rollup generate the types and remove the `build:types`?

Reply:
Yes, it should work